docs/BinSkimRules.md

```diff
@@ -936,3 +936,95 @@ Incremental linking support increases binary size and can reduce runtime perform
 
 ---
 
+## Rule `BA6002.EliminateDuplicateStrings`
+
+### Description
+
+The /GF compiler option, also known as Eliminate Duplicate Strings or String Pooling, will combine identical strings in a program to a single readonly copy. This can significantly reduce binary size for programs with many string resources.
+
+### Messages
+
+#### `Pass`: Pass
+
+'{0}' was compiled with Eliminate Duplicate Strings (/GF) enabled.
+
+#### `Warning`: Warning
+
+'{0}' was compiled without Eliminate Duplicate Strings (/GF) enabled, increasing binary size.  The following modules do not specify that policy: {1}.
+
+#### `InvalidMetadata`: NotApplicable
+
+'{0}' was not evaluated for check '{1}' as the analysis is not relevant based on observed metadata: {2}.
+
+---
+
+## Rule `BA6004.EnableCOMDATFolding`
```

Comment:
>EnableCOMDATFolding [](http://example.com/codeflow?start=16&length=19)

Looks like casing change not applied here? Need to rerun export or manually modify?

Reply:
I think that file might not be part of the .sln so my find/replace missed it.  Will fix.